src/sentry/rules/processing/processor.py

```diff
@@ -297,6 +331,11 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
                 )
                 return
 
+        # If we've reached here we have slow conditions to evaluate
+        if slow_conditions:
```

Comment:
Oh i see

IF the org flag is set, then we save the slow conditions to be enqueued rather than run immediately

can we add that as a comment above?